wger/gym/templates/gym/partial_user_list.html

```diff
@@ -52,3 +63,69 @@
 {% endfor %}
 </tbody>
 </table>
+
+</div>
+
+<!-- table for deactivated members -->
+
+ <div id="inactive_users" class="tabcontent">
+<table class="table table-hover" class="main_member_list">
+    <thead>
+    <tr>
+        {% for key in user_table.keys %}
+            <th>{{ key }}</th>
+        {% endfor %}
+    </tr>
+    </thead>
+    <tbody>
+    {% for current_user in user_table.deactived_users %}
```

Comment:
@Rhytah I am wondering how it's possible to access `deactived_users` yet the key in your context is `deactivated_users`

Reply:
@AGMETEOR I have fixed this typo.